### ISSUE TITLE
Electrum-XVG wallet crashing on OSX

### ISSUE THREAD

**Issue by chapali100** [vergecurrency/VERGE#132]
I'm unable to open the wallet because it keeps terminating with the following message "Electrum-XVG Error".

I'm currently using High Sierra on a Macbook Pro. What am I doing wrong?

**Comment by markwaddle** [comment]
i have experienced this as well. it crashes immediately after starting. it logs a "Month 13 is out of bounds" message to the Console. this appears to be a problem with High Sierra that started occurring Dec 2017. See https://mjtsai.com/blog/2017/12/06/month-13-is-out-of-bounds/
it's possible that is not related at all though and that it is crashing due to another reason.

**Comment by 240mamba** [comment]
Try to delete wallet, then do a fresh download, and move wallet to
applications, worked for me

**Comment by justinvforvendetta** [comment]
yes many people dont seem to know that a .dmg file on osx needs to be dragged into applications.

**Comment by markwaddle** [comment]
why do you assume that i don't have the .app in my /Applications directory? i do. i will try recreating the wallet

**Comment by whit3rabbit** [comment]
I was getting same error when running app on OSX (sierra).  On startup, when creating a wallet, I would get an error box with no msg.  However, I solved issue by running app as sudo:
```
sudo /Applications/Electrum-XVG.app/Contents/MacOS/Electrum-XVG 
```
I created a new wallet and restarted app without sudo.  The app got a little farther (past wallet creation) but would crash when trying to connect to server.  Running as sudo the app started fine.

I'm assuming there is some permission error.
